bower.json

```diff
@@ -12,7 +12,8 @@
     "Sortable": "~1.0.0",
     "animate-sass": "~0.6.2",
     "moment": "~2.8.4",
-    "livestampjs": "~1.1.2"
+    "livestampjs": "~1.1.2",
+    "bootstrap-datepicker": "1.3.1"
```

Comment:
I remember doing this after realising I'd forgotten - but I know why now. The tilde character in OSX is a modifier and if you don't provide a character to modify the ~ will disappear.